package/src/components/Channel/Channel.tsx

```diff
@@ -560,7 +560,7 @@ const ChannelWithContext = <
     handleRetry,
     handleThreadReply,
     hasCameraPicker = isImagePickerAvailable(),
-    hasCommands = true,
+    hasCommands = (channel.getConfig()?.commands ?? []).length > 0,
```

Comment:
This will crash if `channel` is `undefined`

Reply:
This is now handled